cookbook/webpack-encore.rst

```diff
@@ -140,6 +140,10 @@ and run the following command:
 
 Now you should be able to see the outcome in the browser.
 
+.. note::
+
+   When using custom assets with webpack-encore, you need to update the admin build in the public/build/admin directory: :doc:`build-admin-frontend`
```

Comment:
```suggestion
   If webpack encore was accidentally build before changing the `public/build` to `public/website/build`
   it could happen that all your admin assets were removed. Use `git checkout public/build/admin` or the `bin/console sulu:admin:update-build` command to restore the admin build files: :doc:`build-admin-frontend`
```

Reply:
So I would change the text to something like this. Think it is a good hint to add still a note here, for everybody landing into the same situation then you did.